helm/coco-kafka-bridge/app-configs/kafka-bridge_publishing_prod_eu.yaml

```diff
@@ -1,36 +0,0 @@
-# Default values that will be applied to the prod cluster.
-# In pub we shouldn't have any bridges.
-# For now we bridge prod k8s to the coco prod
-bridges:
-## Begin prod bridges
-# todo [sb] remove the bridges when the publishes come only to this cluster (we are 100% live)
-#CMS
-- name: cms-kafka-bridge-pub-coco-prod-eu
```

Comment:
Please do not remove this file, otherwise the default kafka-bridge_publishing.yaml will be applied. Instead, remove all the entries in the bridges list, so only `-bridges:` should remain in this file

Reply:
Ok, I see.